contrib/inventory/ec2.py

```diff
@@ -686,6 +686,8 @@ def include_rds_clusters_by_region(self, region):
                 del c['EarliestRestorableTime']
             if 'LatestRestorableTime' in c:
                 del c['LatestRestorableTime']
+            if 'ClusterCreateTime' in c:
+                del c['ClusterCreateTime']
```

Comment:
Let's fix this properly rather than deleting useful information

https://stackoverflow.com/a/22238613